### ISSUE TITLE
Publish on FDroid

### ISSUE THREAD

**Issue by rugk** [oversecio/oversec_crypto#12]
Could you please publish this on FDroid?

**Comment by Almtesh** [comment]
Oh please, it would be great, please!

**Comment by oversecio** [comment]
I understand and will do some time in the future - but will have to wait a bit more to see how much demand there is for this my technology.

**Comment by rugk** [comment]
Does this really matter? I mean when you publish it on FDroid users may get aware of this new technology and then try it out.

**Comment by Almtesh** [comment]
The same for me. I want to try and use it, but I don't have the Play Store on my phone.

**Comment by oversecio** [comment]
Valid point.

I've just added a link to directly download the APK to the website:

http://www.oversec.io/download/index.html

From the start I've designed Oversec so that it should work without play services installed - 
please let me know though if you encounter any problems...

**Comment by rugk** [comment]
Download over HTTP... :expressionless:

**Comment by Almtesh** [comment]
lol!

**Comment by rugk** [comment]
Actually you can manually use HTTPS there by typing it in, but as @oversecio seems to use Cloudflare that's not secure anyway.

See:
* https://scotthelme.co.uk/cloudflares-great-new-features-and-why-i-wont-use-them/
* https://medium.com/@karthikb351/airtel-is-sniffing-and-censoring-cloudflares-traffic-in-india-and-they-don-t-even-know-it-90935f7f6d98#.f3exdyhwd

**Comment by rugk** [comment]
Yeah that's nice. Anyway this completely unrelated to HTTPS.

So why not host the website on GitHub Pages instead? There [HTTPS is used by default](https://github.com/blog/2186-https-for-github-pages).

**Comment by Luraktinus** [comment]
any news on this after one year?

**Comment by rugk** [comment]
Yeah, besides all this apk/https download discussion I'd still say the most secure, easy and great marketing way would be to add it to F-Droid.

**Comment by Luraktinus** [comment]
the OpenKeybase devs told me that this app is not fully open source... that means this will NOT be added to fdroid because they only want FOSS apps

**Comment by Luraktinus** [comment]
so if this app will not be released as open source, i will remove this from my phone and sugesst to close this issue
